scalanet/src/io/iohk/scalanet/peergroup/InMemoryPeerGroup.scala

```diff
@@ -5,22 +5,23 @@ import monix.reactive.{MulticastStrategy, Observable}
 import monix.reactive.subjects.ConcurrentSubject
 import monix.execution.Scheduler.Implicits.global
 import InMemoryPeerGroup._
+import io.iohk.scalanet.peergroup.PeerGroup.ServerEvent
+import io.iohk.scalanet.peergroup.PeerGroup.ServerEvent.ChannelCreated
 
 import scala.collection.concurrent.TrieMap
-import scala.util.Random
 
 class InMemoryPeerGroup[A, M](address: A)(implicit network: Network[A, M]) extends PeerGroup[A, M] {
 
-  private var status: PeerStatus = PeerStatus.NotInitialized
-  private val channelStream = ConcurrentSubject[InMemoryChannel[A, M]](MulticastStrategy.publish)
-  private val channelsMap: TrieMap[ChannelID, InMemoryChannel[A, M]] = TrieMap()
-  private def receiveMessage(channelID: ChannelID, from: A, msg: M): Unit = {
+  var status: PeerStatus = PeerStatus.NotInitialized
```

Comment:
This is dangerous, why aren't these private anymore? can't you use package private instead?

Reply:
I've made them private to the package.